webapp/src/shared/components/NavigationBar/NavigationBar.tsx

```diff
@@ -52,15 +52,16 @@ export default function NavigationBar() {
   return (
     <nav className="navigation-bar">
       <ul>
-        {navItems.map((item: NavItemContent) => (
-          <li key={item.title}>
-            <NavigationItem
-              iconVariant={item.icon}
-              title={item.title}
-              urlNavigation={item.url}
-            />
-          </li>
-        ))}
+        {navItems &&
```

Comment:
Before there was a brief blinking in and out of existence of a Three itemed navbar in the case of the user being an admin. This should now be fixed.